CMakeLists.txt

```diff
@@ -126,8 +127,10 @@ message("Building common libraries")
 add_subdirectory("libs")
 
 # We always build CorsixTH otherwise we would miss the generated header
```

Comment:
This comment is no longer correct

Reply:
I wish I knew what header the line was referring to.